tests/Feature/ApplicationTest.php

```
@@ -288,1324 +158,1374 @@ public function test_deploy_modern(): void
-    public function test_delete_release(): void
-    {
-        SSH::fake('fake output');
-
-        $this->site->update([
-            'type_data' => [
-                'modern_deployment' => true,
-                'modern_deployment_history' => 10,
-                'modern_deployment_shared_resources' => ['.env'],
+function vitoPestFeatureApplicationTestAssertBroadcastsDeploymentCreated(callable $siteSetup): void
+{
+    SSH::fake('fake output');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Facades\SSH' in method 'vitoPestFeatureApplicationTestAssertBroadcastsDeploymentCreated'. (undefined)

(StaticAccess)